pymesync/pymesync.py

```diff
@@ -527,11 +533,13 @@ def get_users(self, username=None):
             return mock_pymesync.get_users(username)
 
         # Attempt to GET users, then convert the response to a python
-        # dictionary
+        # dictionary. Always returns a list.
         try:
             # Success!
             response = requests.get(url)
-            return self.__response_to_python(response)
+            res_dict = self.__response_to_python(response)
+
+            return [res_dict] if username else res_dict
```

Comment:
Can you comment this so it's more apparent why it's returning `[res_dict] if username`?